modules/virt-adding-virtio-drivers-vm-yaml.adoc

```diff
@@ -43,18 +39,10 @@ volumes:
       image: container-native-virtualization/virtio-win
     name: virtiocontainerdisk
 ----
-<1> {VirtProductName} boots virtual machine disks in the order defined in the
-`VirtualMachine` configuration file. You can either define other disks for the
-virtual machine before the `container-native-virtualization/virtio-win` container disk or use the optional
-`bootOrder` parameter to ensure the virtual machine boots from the correct disk.
- If you specify the `bootOrder` for a disk, it must be specified for all disks
-in the configuration.
-
-. The disk is available once the virtual machine has started:
-** If you add the container disk to a running virtual machine, use
-`oc apply -f <vm.yaml>` in the CLI or reboot the virtual machine for the changes
-to take effect.
-** If the virtual machine is not running, use `virtctl start <vm>`.
-
-After the virtual machine has started, the VirtIO drivers can be installed from
-the attached SATA CD drive.
+<1> {VirtProductName} boots the VM disks in the order defined in the `VirtualMachine` manifest. You can define other VM disks that boot before the `container-native-virtualization/virtio-win` container disk or use the optional `bootOrder` parameter to ensure the VM boots from the correct disk. If you configure the boot order for a disk, you must configure the boot order for the other disks.
```

Comment:
```suggestion
<1> {VirtProductName} boots the VM disks in the order defined in the `VirtualMachine` manifest. You can either define other VM disks that boot before the `container-native-virtualization/virtio-win` container disk, or use the optional `bootOrder` parameter, to ensure the VM boots from the correct disk. If you configure the boot order for a disk, you must configure the boot order for the other disks.
```

Reply:
Out of scope for this PR, but you might consider clarifying this footnote.

Reply:
I will add  "either/or" but the commas are unnecessary, IMO.